Dockerfile

```diff
@@ -1,33 +1,36 @@
-FROM golang:1.17-alpine
+FROM golang:1.17-alpine as base
 MAINTAINER team@pganalyze.com
 
-RUN adduser -D pganalyze pganalyze
 ENV GOPATH /go
 ENV HOME_DIR /home/pganalyze
 ENV CODE_DIR $GOPATH/src/github.com/pganalyze/collector
 
+RUN apk add --no-cache --virtual .build-deps make curl libc-dev gcc git tar \
+  && mkdir -p $HOME_DIR
+
 COPY . $CODE_DIR
 WORKDIR $CODE_DIR
 
-# We run this all in one layer to reduce the resulting image size
-RUN apk add --no-cache --virtual .build-deps make curl libc-dev gcc go git tar \
-  && apk add --no-cache ca-certificates setpriv \
-  && make build_dist_alpine OUTFILE=$HOME_DIR/collector \
-  && rm -rf $GOPATH \
-	&& apk del --purge .build-deps
-
-RUN chown pganalyze:pganalyze $HOME_DIR/collector
-
-RUN mkdir /state
-RUN chown pganalyze:pganalyze /state
-VOLUME ["/state"]
+RUN  make build_dist_alpine OUTFILE=$HOME_DIR/collector 
 
 RUN mkdir -p /usr/share/pganalyze-collector/sslrootcert/
-COPY contrib/sslrootcert/rds-ca-2015-root.pem /usr/share/pganalyze-collector/sslrootcert/
-COPY contrib/sslrootcert/rds-ca-2019-root.pem /usr/share/pganalyze-collector/sslrootcert/
+COPY contrib/sslrootcert/rds-ca-2019-root.pem contrib/sslrootcert/rds-ca-2015-root.pem /usr/share/pganalyze-collector/sslrootcert/
 COPY contrib/docker-entrypoint.sh $HOME_DIR
 RUN chmod +x $HOME_DIR/docker-entrypoint.sh
 
+FROM alpine:3.15 as slim
+
+RUN apk add --no-cache ca-certificates
+
+RUN adduser -D pganalyze pganalyze \ 
+  && mkdir /state  \
+  && chown pganalyze. /state 
```

Comment:
I assume this should be `chown pganalyze:pganalyze`?

Reply:
`chown $user.` is shorthand in linux. either one works, just muscle memory for me to use the shorthand.